src/components/course/data/service.js

```diff
@@ -85,18 +90,24 @@ export default class CourseService {
     return this.cachedAuthenticatedHttpClient.get(url);
   }
 
-  async fetchEnterpriseUserSubsidy() {
+  async fetchEnterpriseUserSubsidy({ offers, catalogList }) {
     // TODO: this will likely be expanded to support codes/offers, by appending to
     // the below array to provide a function that makes the relevant API call(s)
     // for the specified subsidy type.
```

Comment:
nit: Should we remove this TODO comment now? Or keep it in until the future follow-up work is done?

Reply:
also updated this to be current